cmd/cli/operations/update_job_test.go

```diff
@@ -11,39 +11,41 @@ import (
 )
 
 /*
- * filterRunningJobs
+ * filterRunningJobsByName
  */
 func TestFilterRunningJobsShouldReturnAnEmptySliceIfNoJobsWereProvided(t *testing.T) {
 	operator := RealOperator{}
 
-	res := operator.filterRunningJobs([]flink.Job{})
+	res := operator.filterRunningJobsByName([]flink.Job{}, "test")
 
 	assert.Len(t, res, 0)
 }
 
 func TestFilterRunningJobsShouldReturnAnEmptySliceIfNoJobsAreRunning(t *testing.T) {
 	operator := RealOperator{}
 
-	res := operator.filterRunningJobs([]flink.Job{
+	res := operator.filterRunningJobsByName([]flink.Job{
 		flink.Job{
 			Status: "STOPPED",
 		},
-	})
+	}, "test")
 
 	assert.Len(t, res, 0)
 }
 
 func TestFilterRunningJobsShouldReturnTheRunningJobs(t *testing.T) {
```

Comment:
I'd actually like to see this test verify the desired behaviour:

`TestFilterRunningJobsShouldReturnTheRunningJobsMatchingTheJobBaseName` which has 3 running jobs. 2 with the same base name but only 1 of them with the status `RUNNING`. The third job can have a different name with the status `RUNNING`. We expect only 1 result to be returned.

Reply:
Added a new test as requested.